roles/katello_repositories/tasks/staging_repos.yml

```diff
@@ -25,5 +25,3 @@
 - name: 'Add Candlepin repository'
   include_role:
     name: candlepin_repositories
-  when:
-    - candlepin_repositories_version is defined
```

Comment:
I think this change would break pipelines that are still based on Koji.

Reply:
It seems odd that we have this check for candlepin repos but not for pulpcore repos. What am I missing?

I'd also like to avoid creating another value for candlepin_repositories_version that has to be maintained separately, if it can be avoided.

I'll attempt modifying this condition for now.

Reply:
I changed it to `when: candlepin_repositories_version is defined or katello_repositories_staging_source == "stagingyum"` to avoid including the separate candlepin repositories role when using Koji

Reply:
For devel, do you want the default or do you want nightly? e.g. This right now will enable the default which is '4.3'.

Reply:
Summarizing the offline conversation for the record:

We'll go with 4.3 for as default for now.

We'll plan to switch to nightly candlepin as default in the future once the katello integration tests (see: https://theforeman.org/plugins/katello/developers.html#upgrading-candlepin ) can be added to the pipeline.